backend/s3/s3.go

```diff
@@ -3022,6 +3022,14 @@ func (f *Fs) setUploadChunkSize(cs fs.SizeSuffix) (old fs.SizeSuffix, err error)
 	return
 }
 
+func checkCopyCutoff(cs fs.SizeSuffix) error {
+	minCopySize := fs.SizeSuffixBase
+	if cs < minCopySize {
+		return fmt.Errorf("%s is less than %s", cs, minCopySize)
```

Comment:
Can you say what is out of range in the error? Also use %d/%v is preferred, so something like

    return fmt.Errorf("--s3-copy-cutoff is too small (%d is less than %d) ", cs, minCopySize)

Reply:
Actually, there's additional information in outer layer.  
```go
	err = checkCopyCutoff(opt.CopyCutoff)
	if err != nil {
		return nil, fmt.Errorf("s3: copy cutoff: %w", err)
	}
```
Though it doesn't specific which parameter is wrong. I would like to include `--s3-copy-cutoff` here.  
So the output will be:
```
2024/03/14 17:20:47 Failed to create file system for "minipc-s3:/buildscripts/6G": s3: --s3-copy-cutoff: value is too small (0 is less than 1)
```
Is the output clear now?

Reply:
Looks good :-)